prow/config.yaml

```diff
@@ -103,6 +103,11 @@ branch-protection:
             - "Multi-arch build"
             - "e2e-tests / e2e tests (k8s-oldest)"
             - "e2e-tests / e2e tests (k8s-plus-one)"
+        triggers:
+          required_status_checks:
+            contexts:
+            - "lint / lint"
```

Comment:
I think it's gonna be  just "lint" isn't it ?
Same for Integration Tests.

Reply:
I am not sure. I have changed to that. Let's merge this.